app/controllers/image_uploads_controller.rb

```diff
@@ -46,6 +53,15 @@ def cloudinary_link(uploaders)
 
   private
 
+  def rate_limit!
```

Comment:
I wonder if we could move all these `rate_limit!` methods into the `RateLimitChecker` since it seems like their logic is all very similar.

Reply:
hahaha @mstruve I thought about it but I didn't want to revolutionize too much. I'll make a note for another PR

Reply:
![](https://media3.giphy.com/media/l2SpV7Qi9GXY6CNwc/giphy.gif)